### PR TITLE
Update FFmpeg log

### PR DESCRIPTION
**Pull request by Artiume**
**Changes**

test

**Issues**

### CODE REVIEW

File: Jellyfin.Api/Helpers/TranscodingJobHelper.cs [hunk 1]
```diff
@@ -544,18 +544,18 @@ private void DeleteHlsPartialStreamFiles(string outputFilePath)
...
             {
                 logFilePrefix = EncodingHelper.IsCopyCodec(state.OutputAudioCodec)
-                    ? "ffmpeg-remux"
-                    : "ffmpeg-directstream";
+                    ? "FFmpeg.Remux-"
+                    : "FFmpeg.DirectStream-";
             }
 
-            var logFilePath = Path.Combine(_serverConfigurationManager.ApplicationPaths.LogDirectoryPath, logFilePrefix + "-" + Guid.NewGuid() + ".txt");
+            var logFilePath = Path.Combine(_serverConfigurationManager.ApplicationPaths.LogDirectoryPath, logFilePrefix + DateTime.Now.ToString("yyyy-MM-dd_HH-mm-ss") + "_" + state.Request.MediaSourceId + ".log");
```
Comment (crobibero) [inline Jellyfin.Api/Helpers/TranscodingJobHelper.cs:556]: I might still append Guid.NewGuid() in the extremely rare case that two ffmpeg streams start at the same time for the same media.

Comment (Artiume) [reply Jellyfin.Api/Helpers/TranscodingJobHelper.cs:556]: syncplay comes in mind

**Comment by crobibero** [comment]
Build failing with 
```
/home/vsts/work/1/s/Jellyfin.Api/Helpers/TranscodingJobHelper.cs(556,123): error CA1305: The behavior of 'DateTime.ToString(string)' could vary based on the current user's locale settings. Replace this call in 'TranscodingJobHelper.StartFfMpeg(StreamState, string, string, HttpRequest, TranscodingJobType, CancellationTokenSource, [string])' with a call to 'DateTime.ToString(string, IFormatProvider)'. [/home/vsts/work/1/s/Jellyfin.Api/Jellyfin.Api.csproj]
```

**Comment by Artiume** [comment]
> Build failing with
> 
> ```
> /home/vsts/work/1/s/Jellyfin.Api/Helpers/TranscodingJobHelper.cs(556,123): error CA1305: The behavior of 'DateTime.ToString(string)' could vary based on the current user's locale settings. Replace this call in 'TranscodingJobHelper.StartFfMpeg(StreamState, string, string, HttpRequest, TranscodingJobType, CancellationTokenSource, [string])' with a call to 'DateTime.ToString(string, IFormatProvider)'. [/home/vsts/work/1/s/Jellyfin.Api/Jellyfin.Api.csproj]
> ```

what do i do exactly?

**Comment by oddstr13** [comment]
Invariant thingy afaik, found this;
```csharp
            var requestReceptionTime = DateTime.UtcNow.ToUniversalTime().ToString("o", DateTimeFormatInfo.InvariantInfo);
```

File: Jellyfin.Api/Helpers/TranscodingJobHelper.cs (hunk 1 above)
Comment (crobibero) [inline Jellyfin.Api/Helpers/TranscodingJobHelper.cs:556]:
```suggestion
            var logFilePath = Path.Combine(_serverConfigurationManager.ApplicationPaths.LogDirectoryPath, logFilePrefix + DateTime.Now.ToString("yyyy-MM-dd_HH-mm-ss", CultureInfo.InvariantCulture) + "_" + state.Request.MediaSourceId + "_" + Guid.NewGuid().ToString("N") + ".log");
```

**Comment by Artiume** [comment]
test

With attaching media GUID and a random GUID. the file length is really long. is there any way to create a shorter GUID like 8 characters long?

**Comment by Artiume** [comment]
`var logFilePath = Path.Combine(_serverConfigurationManager.ApplicationPaths.LogDirectoryPath, logFilePrefix + DateTime.Now.ToString("yyyy-MM-dd_HH-mm-ss", CultureInfo.InvariantCulture) + "_" + state.Request.MediaSourceId + "_" + Guid.NewGuid().ToString("N").Substring(0, 8) + ".log");`

test

File: Jellyfin.Api/Helpers/TranscodingJobHelper.cs [hunk 2]
```diff
@@ -544,18 +544,18 @@ private void DeleteHlsPartialStreamFiles(string outputFilePath)
...
             {
                 logFilePrefix = EncodingHelper.IsCopyCodec(state.OutputAudioCodec)
-                    ? "ffmpeg-remux"
-                    : "ffmpeg-directstream";
+                    ? "FFmpeg.Remux-"
+                    : "FFmpeg.DirectStream-";
             }
 
-            var logFilePath = Path.Combine(_serverConfigurationManager.ApplicationPaths.LogDirectoryPath, logFilePrefix + "-" + Guid.NewGuid() + ".txt");
+            var logFilePath = Path.Combine(_serverConfigurationManager.ApplicationPaths.LogDirectoryPath, logFilePrefix + DateTime.Now.ToString("yyyy-MM-dd_HH-mm-ss", CultureInfo.InvariantCulture) + "_" + state.Request.MediaSourceId + "_" + Guid.NewGuid().ToString("N").Substring(0, 8) + ".log");
```
Comment (crobibero) [inline Jellyfin.Api/Helpers/TranscodingJobHelper.cs:556]:
```suggestion
            var logFilePath = Path.Combine(
                _serverConfigurationManager.ApplicationPaths.LogDirectoryPath,
                $"{logFilePrefix}{DateTime.Now:yyyy-MM-dd_HH-mm-ss}_{state.Request.MediaSourceId}_{Guid.NewGuid().ToString()[..8]}.log");
```

Shorter way of doing the same thing

**Review by crobibero — APPROVED** [review]

**Review by oddstr13 — APPROVED** [review]

**Pull request merged**